java/cuvs-java/src/main/java/com/nvidia/cuvs/CagraQuery.java

```diff
@@ -31,6 +32,8 @@ public class CagraQuery {
   private List<Integer> mapping;
   private float[][] queryVectors;
   private int topK;
+  private BitSet[] prefilters;
```

Comment:
I thought Cagra doesn't support per query prefilters. Why do we have an array of prefilters here? Did I misunderstand the first part?